utils/src/ooinstall/oo_config.py

```diff
@@ -325,6 +326,10 @@ def _set_defaults(self):
         self.settings['ansible_inventory_path'] = \
             '{}/hosts'.format(os.path.dirname(self.config_path))
 
+        # pylint: disable=consider-iterating-dictionary
+        # Disabled because we shouldn't alter the container we're
+        # iterating over
+        #
         # clean up any empty sets
         for setting in self.settings.keys():
```

Comment:
In python3 this would return an iterator. I haven't checked out python3 compatibility and not sure we even care, but this is a potential issue there.